testing/python-poetry/tests/BUILD.bazel

```diff
@@ -0,0 +1,18 @@
+load("@nixpkgs_python_toolchain//:defs.bzl", "interpreter")
+load("@nix_pip//:requirements.bzl", "requirement")
```

Comment:
Should it perhaps be called `nix_poetry` instead? Since it's based on poetry instead of pip...

Apart from that I like the similarity to the [rules_python pip API](https://github.com/bazelbuild/rules_python#installing-third_party-packages). That should help make adoption easy.